tests/test_counter.py

```diff
@@ -0,0 +1,13 @@
+"""This module test the counter.py module regarding if statements"""
+from castanet import counter
+from castanet import generate_trees as generator
+
+def test_identifing_ifs():
+    """Check that match_ifstatements identifies all of the if-statements in a directory"""
+    directory = "./hello"
```

Comment:
Please create a test case that uses another directory other than just `hello`. This can be a file that you create on your own that is placed in the `tests` folder.